src/Sylius/Behat/Context/Ui/Shop/AddressBookContext.php

```diff
@@ -254,11 +262,11 @@ public function iShouldBeNotifiedThatTheProvinceNeedsToBeSpecified()
     }
 
     /**
-     * @Then /^I should be notified about (\d+) errors$/
+     * @Then /^I should be notified about errors$/
      */
-    public function iShouldBeNotifiedAboutErrors($expectedCount)
+    public function iShouldBeNotifiedAboutErrors(): void
     {
-        Assert::same($this->addressBookCreatePage->countValidationMessages(), (int) $expectedCount);
+        Assert::true($this->addressBookCreatePage->countValidationMessages() > 0);
```

Comment:
Previously we knew at least the amount of errors presented. Now we only know, that at least one happened. I'm not convinced about this change.